### PR TITLE
[Documentation] Update all READMEs to be reflected in the github wiki

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

Update all READMes to have a tag so they are reflected in the [Github Wiki](https://github.com/pokt-network/pocket/wiki).

## Issue

NA. Follow up to #419

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Major breaking change
- [x] Documentation
- [ ] Other 

## List of changes

- Added a link to the [Github Wiki](https://github.com/pokt-network/pocket/wiki) in the learning guide
- Update all READMEs that were missing a github wiki tag
- Bonus: Added a link to Alin's blog

## Testing

- [x] `make develop_test`
- [ ] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

### CODE REVIEW

File: app/client/doc/CHANGELOG.md [hunk 1]
```diff
@@ -71,3 +71,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Functionally mocked a keybase in the form of a json file (default: pk.json) that will contain the privatekey
 - CLI calling RPC via generated client
 - Default configuration handling/overrides
+
+<!-- GITHUB_WIKI: changelog/app -->
```
Comment (bryanchriswhite) [inline app/client/doc/CHANGELOG.md:75]:
```suggestion
<!-- GITHUB_WIKI: changelog/client -->
```

What happens in the case where multiple tags use the same path? It looks like this changelog and app/pocket/doc/CHANGELOG.md are doing just that.

File: docs/learning/README.md [hunk 2]
```diff
@@ -59,7 +59,11 @@ This is a general set of steps we have found to help new core team members onboa
 
    - It is a proprietary algorithm that you are unlikely to ever need to touch/modify, but it is fun & interesting to reason about and understand.
 
-7. **Eventually**: Over time, **and not all at once**, you can start making your way through and updating the [V1 specifications](https://github.com/pokt-network/pocket-network-protocol). _(15-30 hours)_
+7. Find all of our documentation on the [Github Wiki](https://github.com/pokt-network/pocket/wiki).
+
+   - The wiki mirrors all of the READMEs in our codebase so it's easy to find and navigate!
```
Comment (bryanchriswhite) [inline docs/learning/README.md:64]: Nice! :sunglasses:

File: docs/releases/README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # Building & Versioning
 
-- [Building & Versioning](#building--versioning)
+- [Building \& Versioning](#building--versioning)
```
Comment (bryanchriswhite) [inline docs/releases/README.md:3]: Nice catch!

Comment (Olshansk) [reply docs/releases/README.md:3]: Appreciate it but it wasn't me. The extension automatically did it for me on save: https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

File: docs/learning/README.md [hunk 4]
```diff
@@ -73,18 +77,18 @@ This is a general set of steps we have found to help new core team members onboa
          - [Consensus](https://docs.google.com/presentation/d/18CtSxxLLHY1N7HEJtja633mVF1_a9blaE2fe2-WgGAo/edit)
    -->
 
-8. Start getting acquainted with the code structure by looking at the [docs on the shared architecture](https://github.com/pokt-network/pocket/tree/main/shared). _(1 hour)_
+9. Start getting acquainted with the code structure by looking at the [docs on the shared architecture](https://github.com/pokt-network/pocket/tree/main/shared). _(1 hour)_
```
[the remark this answers is not in the document]
Comment (Olshansk) [reply docs/learning/README.md:80]: At some point, we'll create a markdown lint.rc so it's the same for everyone: https://github.com/pokt-network/pocket/issues/257

Most of these NITS were down on save by VSCode

File: docs/learning/README.md (hunk 2 above)
Comment (Olshansk) [reply docs/learning/README.md:64]: Agreed

File: app/client/doc/CHANGELOG.md (hunk 1 above)
Comment (Olshansk) [reply app/client/doc/CHANGELOG.md:75]: I'm adding the following TODO in `wiki_sync.py`:

```
# TECHDEBT: Need to verify and exit the CI with a non-zero code if there is a collision on `WIKI_TAG`
```

Comment (Olshansk) [reply app/client/doc/CHANGELOG.md:75]: Nvm, it was 3 lines of code:

```py
        if wiki_path_format in seen_wiki_paths:
            raise ValueError(f"Duplicate wiki path: {wiki_path_format}")
        seen_wiki_paths.add(wiki_path_format)
```

**Comment by Olshansk** [comment]
@bryanchriswhite Replied to your comments and updated the changelogs. PTAL when you have a chance

**Review by bryanchriswhite — APPROVED** [review]

**Pull request merged**